tests/xfstests/enable_kdump.pm

```diff
@@ -28,7 +28,7 @@ sub run {
     # Also panic when softlockup
     # workaround bsc#1104778, skip s390x in 12SP4
     assert_script_run('echo "kernel.softlockup_panic = 1" >> /etc/sysctl.conf');
-    my $output = script_output('sysctl -p');
+    my $output = script_output('sysctl -p', 10, type_command => 1, proceed_on_failure => 1);
```

Comment:
You don't actually need to put `type_command => 1` as it's default. Would you mind to remove it? No new VR is required.